javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,68 @@
+//Age calculator
+
+let yearOfBirth = 1995
+let yearFuture = 2027
+let age = yearFuture - yearOfBirth
+console.log("You will be " + age + " years old in " + yearFuture)
+
+//Goodboy-oldboy
+
+let dogYearOfbirth = 2018;
+let dogYearFuture = 2027;
+let dogYear = yearFuture - yearOfBirth;
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+    let dogInDogYears = dogYear * 7;
+    console.log("Your dog will be " + dogInDogYears + " dog years old in 2027");
```

Comment:
You can make calculations inside the `console.log`, so you'll have less code:
`console.log("Your dog will be " + dogYear * 7+ " dog years old in 2027");`